src/main/external-resources/renderers/Samsung-9series.conf

```diff
@@ -60,16 +60,19 @@ Supported = f:webm     v:vp8|vp9                                           a:vor
 Supported = f:wmv|asf  v:mp4|vc1|vp6|wmv                                   a:wma                                               m:video/x-ms-wmv
 
 # Supported audio formats:
-Supported = f:m4a|alac       m:audio/mp4
 Supported = f:aiff           m:audio/L16
 Supported = f:ape            m:audio/ape
 Supported = f:flac           m:audio/x-flac        n:2
+Supported = f:m4a|alac       m:audio/mp4
 Supported = f:mp3            m:audio/mpeg
-Supported = f:ogg            m:audio/ogg
+Supported = f:oga            m:audio/ogg
 Supported = f:vorbis         m:audio/vorbis        n:2
 Supported = f:wma            m:audio/x-ms-wma
 Supported = f:wav            m:audio/wav
 
+# Supported image formats:
+#Supported = f:mpo    m:image/mpo
```

Comment:
No problem to leave comments but the format of the comment was a bit confusing to me. In other renderer configs we have left notes near the top or bottom for reference, maybe change it to that?